data/json/npcs/missiondef.json

```diff
@@ -711,5 +711,26 @@
       "success_lie": "OK, then hand it over.",
       "failure": "Shit happens."
     }
+  },
+  {
+    "id": "MISSION_GET_CITY_COP_MOMENTO",
+    "type": "mission_definition",
+    "name": "Break into armory to retrieve family photo",
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 2,
+    "value": 800,
+    "item": "family_photo",
+    "origins": [ "ORIGIN_ANY_NPC" ],
+    "dialogue": {
+      "describe": "I left the only photo I have from my family in the armory. I don't have the code, can you get in?",
+      "offer": "I left the only photo I have from my family in the armory. I don't have the code, can you get in?",
```

Comment:
Maybe that should be "Hey, are you any good with locks?  I've left something in the armory downstairs."

Reply:
armory is upstairs and accessible through terminal.  I can change this but not to locks & downstairs.

Reply:
Whatever works for you.  Just the describe and offer are usually different text.

Reply:
ah, understood.